code/modules/power/powernet.dm

```diff
@@ -331,6 +331,9 @@ var/global/powernets_broke = 0
 		if(H.gloves)
 			var/obj/item/clothing/gloves/G = H.gloves
 
+			if(G.name == "insulted gloves")
```

Comment:
type check here would be better, even with the == since its less intensive but up to you to use that instead of istype()

Reply:
This type of check isn't too bad because strings are all stored as references on the string table... That said, this prevents renaming gloves for meme purposes. And hand labelers will silence the gloves for good.

This whole setup could be added as a general variable, such as var/insults_on_shock, to obj/item/clothing/gloves for adminbus purposes. Then you can simply check the variable here. Benefits? Admins get a new button to mess with players. Drawbacks? New meme variable on all gloves.

Reply:
speaking of meme variables, why does every fucking atom have a list of armor and a bunch of other stuff they never use?
if i recall right lummox said something about how retarded that is once